test/ufuzz.js

```diff
@@ -582,6 +582,14 @@ function _createExpression(recurmax, noComma, stmtDepth, canThrow) {
       case 30:
         return '(' + createObjectLiteral(recurmax, COMMA_OK, stmtDepth, canThrow) + '.' +
             SAFE_KEYS[rng(SAFE_KEYS.length)] + ") ";
+      case 31:
+        var name = VAR_NAMES[rng(VAR_NAMES.length)];
+        return '(' + name + ' && ' + name + '[' +
+            createExpression(recurmax, COMMA_OK, stmtDepth, canThrow) + "]) ";
+      case 32:
+        var name = VAR_NAMES[rng(VAR_NAMES.length)];
```

Comment:
`createVarName()` [exists](https://github.com/mishoo/UglifyJS2/blob/1f1fccc45df9fdc8a77d2ee3f1ae36f029b31ee9/test/ufuzz.js#L683-L692), but it has a nasty habbit of creating _new_ (thus in this case, undeclared) variables.

There is existing use case of bypassing the creation by [using `VAR_NAMES` directly](https://github.com/mishoo/UglifyJS2/blob/1f1fccc45df9fdc8a77d2ee3f1ae36f029b31ee9/test/ufuzz.js#L643) as well.

Or do you want me to make a new say `getVarName()` with this just to make it look prettier?